INSTALLATION.md

```diff
@@ -13,6 +13,52 @@ cilium install --version -service-mesh:v1.11.0-beta.1 --config enable-envoy-conf
 cilium hubble enable
 ```
 
+Alternatively, you can use Helm to install Cilium Service Mesh. For example, to install on GKE:
+
+```
+# Specify your cluster name and zone.
+CLUSTER_NAME=my-cluster
+CLUSTER_ZONE=us-west2-a
+
+NATIVE_CIDR="$(gcloud container clusters describe $CLUSTER_NAME --zone $CLUSTER_ZONE --format 'value(clusterIpv4Cidr)')"
+git clone git@github.com:cilium/cilium.git
+cd cilium
+git checkout origin/beta/service-mesh
+cat > service-mesh-values.yaml <<EOF
+image:
+  repository: quay.io/cilium/cilium-service-mesh
+  tag: v1.11.0-beta.1
+  useDigest: false
+operator:
+  image:
+    repository: quay.io/cilium/operator
+    tag: v1.11.0-beta.1
+    useDigest: false
+    suffix: "-service-mesh"
+kubeProxyReplacement: probe
```

Comment:
Can we avoid this? Is KPR required? If yes => `strict`; if no => `disabled` (default).

Reply:
not sure, cc @jrajahalme :thinking:

Reply:
We're using `--kube-proxy-replacement=probe` in the CLI version so maybe we could live with this?